cypress/integration/preserve-auth0-cookies-once.spec.js

```diff
@@ -1,30 +1,50 @@
-/**
- * Cypress automatically clears all cookies before each test to prevent state
- * from building up, that's why we test preserving cookies here.
- */
-context('Cookie', () => {
-  before(() => {
-    cy.login();
-  });
+const tests = [
+  {
+    title: 'Cookie',
+    credentials: undefined,
+  },
+  {
+    title: 'Split Cookie',
+    credentials: {
+      username: Cypress.env('auth0SplitCookieUsername'),
+      password: Cypress.env('auth0SplitCookiePassword'),
+    },
+  },
+];
 
-  beforeEach(() => {
-    cy.preserveAuth0CookiesOnce();
-  });
+for (const { title, credentials } of tests) {
+  /**
+   * Cypress automatically clears all cookies before each test to prevent state
+   * from building up, that's why we test preserving cookies here.
+   */
+  context(title, () => {
```

Comment:
```suggestion
  context('Cookie Handling', () => {
```

Reply:
Once you take care of https://github.com/sir-dunxalot/cypress-nextjs-auth0/pull/51/files#r817687714, this will be obsolete.